docs/content/config/setup.sh.md

```diff
@@ -66,7 +66,7 @@ OPTIONS
 
         -p PATH
             Provides the config folder path. The default is
-            /home/maxwell/Documents/github/docker-mailserver/config/
+            /${HOME}/Documents/github/docker-mailserver/config/
```

Comment:
This is still a personalized "default" isn't it?
Is this somewhere in the code or just a "hint" on usage?

I would prefer something like `$(pwd)/config` **but only if** one maps the `./config` in the volumes section.

The current state is a bit misleading imho. Or do i overlook smth?

Reply:
You are correct, I did not think about this. The solution using `${PWD}` is better suited, but I guess at best, we should describe it with text (only)?

Reply:
Text only would be the "most correct" form for every setup imho. Maybe we have a link to the docs at this point (which need to be covered on changes so not optimal as well)

Reply:
I'd use plain old text only. This way, we  can avoid confusion and the mistakes spotted above.

Reply:
Lets that that the user is running the command in `<example>` and then after print out the result with a full path